app/models/schools/configuration.rb

```diff
@@ -1,17 +1,22 @@
 module Schools
   class Configuration
     class Discord
-      attr_accessor :bot_token, :server_id, :default_role_ids
+      attr_accessor :bot_token, :server_id, :bot_user_id
 
       def initialize(school)
-        @discord = school.configuration["discord"].presence || {}
+        @school = school
+        @discord = @school.configuration["discord"].presence || {}
         @bot_token = @discord["bot_token"]
         @server_id = @discord["server_id"]
-        @default_role_ids = @discord["default_role_ids"]
+        @bot_user_id = @discord["bot_user_id"]
+      end
+
+      def default_role_ids
+        @school.discord_roles.where(default: true).pluck(:discord_id)
       end
```

Comment:
It appears that this method is used only in `Discord::SyncProfileService`. Wouldn't it be clearer to remove this method from here, and to replace it with a `School#default_discord_role_ids`?